ow_faults/cfg/Faults.cfg

```diff
@@ -58,5 +51,10 @@ gen.add("hand_yaw_torque_sensor_failure", bool_t,   0, "Hand yaw torque sensor f
 gen.add("scoop_yaw_encoder_failure",       bool_t,   0, "Scoop yaw encoder failure",       False)
 gen.add("scoop_yaw_torque_sensor_failure", bool_t,   0, "Scoop yaw torque sensor failure", False)
 
+# POWER FAULTS
+gen.add("low_state_of_charge_power_failure", bool_t,   0, "Low state of charge power failure", False)
+gen.add("instantaneous_capacity_loss_power_failure", bool_t,   0, "Instantaneous capcity loss power failure", False)
+gen.add("thermal_power_failure", bool_t,   0, "Thermal power failure", False)
+
 exit(gen.generate(PACKAGE, "faults", "Faults"))
 
```

Comment:
Minor general comment about this file.  There are many commented-out lines.  Add a comment saying that these are potential additions, if that's what they are.  If they are obsolete, just delete them.

Reply:
ok! I think these were some of Terry's old code so I didn't want to touch them. Will just delete them since I think they're just old.